src/docs/app-monitoring/sre-guidelines-for-platform-shared-services.md

```diff
@@ -0,0 +1,201 @@
+
+Determining what you want to assure your customers is all about deciding how dependable you want your service to be based on your customers' expectations. For instance, if your SLA specifies that customers should receive a response to their requests within 300 milliseconds, your SLO might set a goal for response times to be within 200 milliseconds. Choosing the right SLO can be a challenge.
+
+Once we establish an SLA that we know will keep our users satisfied, the SLO becomes the minimum commitment we make. As a result, it's in our best interest to identify and address any issues before they breach our SLA, allowing us time to fix them. Breaking this commitment often carries consequences.
+
+## Connections and differences
+- **Connection**: SLIs provide the data to measure SLOs. SLOs define the targets that form the basis of SLAs.
+- **Difference**: SLIs are data-focused and used internally, SLOs are internal targets that guide service performance, and SLAs are external contracts that formalize service expectations and penalties.
+
+Once again, let's use the Platform Product Registry as an example, and we'll consider a continuous 30 day window periods:
+
+- Retrieving all product information on the dashboard should take less than 5 seconds
+- Retrieving information for 30 products on the dashboard should take less than 2 seconds
+- Web, API, and database services should be operational 99.5% of the time
+- The database should have a backup created every 30 minutes
+- Automation jobs for OCP Project Set change requests should be completed within 40 minutes
+
+In case we need to adjust these objectives or schedule maintenance windows, we must communicate the reasons for doing so in the [#internal-devops-registry](https://chat.developer.gov.bc.ca/group/internal-devops-registry) Rocket.Chat channel.
+
+**The frequency of backups** such as every 30 minutes, directly links to a system's Recovery Point Objective (RPO), which serves as a critical metric in disaster recovery and business continuity planning. RPO establishes the maximum allowable data loss, measured in time. 
+
+It addresses the question: "How much data can we lose before it starts affecting our business operations?" The reason for configuring it at a 30-minute interval is due to the unique nature of the Platform Product Registry app. 
+
+The Platform Product Registry typically "backs up" most of its crucial data in GitHub repositories, and the provisioner usually processes each request in less than 30 minutes.
+
+**Why is this important?**
```

Comment:
it feels a bit unrelated here. You setup a 30min RPO because of the SLO `The database should have a backup created every 30 minutes`?